website/src/pages/downloads/ProjectCard.js

```diff
@@ -112,12 +156,47 @@ const ProjectCard = (props) => {
           Release Date ·{" "}
           <span className="downloads-versioninfo-span">{releaseDate}</span>
         </VersionInfo>
-        <div>
+        
+         <ButtonRow>
+          <LTSCard>
+          <LTSButton />
+          <LTSDropdown
+            isLTSDropdownOpen={isLTSDropdownOpen}
+            setIsLTSDropdownOpen={setIsLTSDropdownOpen}
+          >
+            <DropdownItem
+              className="download-dropdown-item"
+              href={`https://www.apache.org/dyn/closer.cgi/apisix/2.10.3/apache-apisix-2.10.3-src.tgz`}
```

Comment:
As the version is updated we may need to be more flexible with this link?🤔

Reply:
Yep,maybe the version info can be put in the json file which include all version info